framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql

```diff
@@ -0,0 +1,5 @@
+set `exec.return_result_set_for_ddl` = false;
+--@test
+set `store.format` = 'json';
```

Comment:
Not sure about what this is testing. May be the CTAS is missing?

Reply:
This is not a DDL statement, but it is also affected by the option tested.

Reply:
So the SQL statement being tested here is the `set option=value`. Got it thanks.